compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1958,57 +1958,69 @@ OMR::ARM64::TreeEvaluator::vstoreiEvaluator(TR::Node *node, TR::CodeGenerator *c
    return TR::TreeEvaluator::vstoreEvaluator(node, cg);
    }
 
+typedef TR::Register *(*reductionEvaluatorHelper)(TR::Node *node, TR::DataType et, TR::Register *resReg, TR::Register *srcReg, TR::CodeGenerator *cg);
 /**
- * @brief A helper function for generating instuction sequence for reduction add operations for vectors of integer elements.
+ * @brief Helper functions for generating instruction sequence for reduction operations
```

Comment:
This has to be "A helper function for ..." instead of "Helper functions for ..."

Reply:
Fixed it.